docs/services/motion/_index.md

```diff
@@ -0,0 +1,466 @@
+
+The volumes associated with all configured robot components (local and remote) will be taken into account for each request to ensure that the robot does not collide with itself.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `component_name` ([ResourceName](https://python.viam.dev/autoapi/viam/gen/common/v1/common_pb2/index.html#viam.gen.common.v1.common_pb2.ResourceName)): Name of the piece of the robot that should arrive at the destination.
+  Note that this is solved such that the distal end of the component arrives at the destination.
+  For example, when moving a robotic arm, the piece that will arrive at the destination is the end effector attachment point, not the base of the arm.
+
+- `destination` ([PoseInFrame](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.PoseInFrame)):
+  Describes where the `component_name` should end up.
+  Can be any pose, from the perspective of any component whose location is configured as a [`frame`](../frame-system/).
+
+  Note that the destination pose is relative to the distal end of the specified frame.
+  This means that if the `destination` is the same as the `component_name` frame, for example an arm's frame, then a pose of {X: 10, Y: 0, Z: 0} will move that arm’s end effector by 10 mm in the local X direction.
+
+- `world_state` ([WorldState](https://python.viam.dev/autoapi/viam/proto/common/index.html#viam.proto.common.WorldState)) (*optional*): Data structure specifying information about the world around the robot.
+  Used to augment the motion solving process.
+  `world_state` includes obstacles and transforms:
+  - **Obstacles**: Geometries located at a pose relative to some frame.
+    When solving a motion plan with movable frames that contain inherent geometries, the solved path is constrained such that none of those inherent geometries intersect with the obstacles.
+    Important considerations:
+    - If a motion begins with a component already in collision with an obstacle, collisions between that specific component and that obstacle will not be checked.
+    - The Motion Service assumes that obstacles with mobile parents move along with their parents while solving.
+      This ensures that obstacles that are temporarily attached to moving components do not cause collisions during movement.
+    - Geometries are "part of" their frame, rather than at the distal end of the frame.
+      Their poses are relative to the *origin* of the specified frame.
+      A geometry associated with the frame of an arm with a pose of {X: 0, Y: 0, Z: -10} will be interpreted as being 10mm below the base of the arm, not 10mm below the end effector.
+      This is different from `destination` and `component_name`, where poses are relative to the distal end of a frame.
+  - **Transforms**: These are a list of `PoseInFrame` messages that specify arbitrary other transformations that will be ephemerally added to the frame system at solve time.
```

Comment:
@JessamyT No update to this line?

Reply:
Yeah...can "arbitrary" come out? And I guess I should add another sentence like 
```suggestion
  - **Transforms**: These are a list of `PoseInFrame` messages that specify other transformations to temporarily add to the frame system at solve time.
  Transforms can be used to account for geometries that are attached to the robot but not configured as robot components.
  For example, you could use a transform to represent the volume of a marker held in your robot's gripper.
  These transformations will not be added to the config or carried into later processes.
```